test/language/expressions/arrow-function/arrow/arrayfunctions_2.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: arrow function array manipulations
+es6id: 14.2
+---*/
+
+var arr = ["check", 1];
+ arr = arr.map(v => v + 1);
+assert.sameValue(arr[0], "check1", "the arr[0] value should be 'check1'.");
+assert.sameValue(arr[1], 2, "the arr[1] value should be 2.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/cannot-override-this-with-thisArg.js